proposals/lip-0052.md

```diff
@@ -133,10 +133,10 @@ The following constants are used throughout the document:
 | `NFT_NOT_LOCKED` | string | `MODULE_NAME_NFT` |
 | `ALL_SUPPORTED_NFTS_KEY` | bytes | `EMPTY_BYTES` |
 | **NFT Store Constants** | | |
-| `SUBSTORE_PREFIX_NFT` | bytes | 0x00 00 |
-| `SUBSTORE_PREFIX_USER` | bytes | 0x80 00 |
-| `SUBSTORE_PREFIX_ESCROW` | bytes | 0xc0 00 |
-| `SUBSTORE_PREFIX_SUPPORTED_NFTS` | bytes | 0xd0 00 |
+| `SUBSTORE_PREFIX_NFT` | bytes | 0x8000 |
+| `SUBSTORE_PREFIX_USER` | bytes | 0x4000 |
+| `SUBSTORE_PREFIX_ESCROW` | bytes | 0xc000 |
+| `SUBSTORE_PREFIX_SUPPORTED_NFTS` | bytes | 0x2000 |
```

Comment:
Are these the store prefixes in the implementation? Why is the `0x0000` prefix missing?  Could you provide a link to the implementation?

Reply:
Yes. Here is the [implementation]( https://github.com/LiskHQ/lisk-sdk/blob/feature/6917-implement-nft-module/framework/src/modules/nft/module.ts#L119-L123). I do not know the reasoning why `0x0000` is missing in the implementation. Btw you can find all the relevant sdk implementation links in the [issue](https://github.com/LiskHQ/lips/issues/406).